docs/csharp/language-reference/compiler-messages/cs0592.md

```diff
@@ -39,4 +39,4 @@ public class A
 
 ## See also
 
-- [Attributes](/dotnet/csharp/advanced-concepts/reflection-and-attributes)
+- [Attributes](/dotnet/csharp/advanced-topics/reflections-and-attributes)
```

Comment:
reflection shouldn't be plural here, should it?

Reply:
nope. One more time....